### PR TITLE
Add basic user actions (sign up, sign in, sign out)

### PR DESCRIPTION
**Pull request by miatrinity**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,17 +1,94 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>RouteinesInit</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WhatsNext</title>
```
Comment (MiaSinek) [inline app/views/layouts/application.html.erb]: The title is Routines NOT WhatsNext

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,17 +1,94 @@
...
+                    <i class="fa fa-sign-out"></i>
+                  </span>
+                  Sign Out
+                </div>
+              <% end %>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
```
Comment (MiaSinek) [inline app/views/layouts/application.html.erb]: Extract navigation into partial

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,17 +1,94 @@
...
+    <div class="has-text-grey has-text-centered is-size-7">
+      <div>
+        Icons made by <a href="https://www.flaticon.com/authors/prettycons" title="prettycons">prettycons</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>
+      </div>
+      <div>
+        Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>
+      </div>
+      <br>
+    </div>
+  </footer>
```
Comment (MiaSinek) [inline app/views/layouts/application.html.erb]: Extract footer into a partial

File: app/views/layouts/sign_in.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WhatsNext</title>
```
Comment (MiaSinek) [inline app/views/layouts/sign_in.html.erb]: The title is routines NOT WhatsNext

**Review by MiaSinek — CHANGES_REQUESTED** [review]
Remove images not used in the design.
See my other comments in the code.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (miatrinity) [reply app/views/layouts/application.html.erb]: Done

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (miatrinity) [reply app/views/layouts/application.html.erb]: Done

File: app/views/layouts/sign_in.html.erb (hunk 4 above)
Comment (miatrinity) [reply app/views/layouts/sign_in.html.erb]: Done

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (MiaSinek) [reply app/views/layouts/application.html.erb]: Ehm... something doesn't add up here. Please come back to this!

**Review by MiaSinek — APPROVED** [review]
LGTM

**Pull request merged**